Sources/Ignite/Elements/Dropdown.swift

```diff
@@ -85,9 +85,12 @@ public struct Dropdown: BlockElement, NavigationItem {
     public func render(context: PublishingContext) -> String {
         Group(isTransparent: isNavigationItem) {
             if isNavigationItem {
+                let hasActiveItem = items.reduce(false) { partialResult, item in
+                    partialResult || context.currentRenderingPath == (item as? Link)?.url
+                }
```

Comment:
No need to iterate all items.
```suggestion
                let hasActiveItem = items.contains { context.currentRenderingPath == ($0 as? Link)?.url  }
```

Reply:
Yes that's a nicer solution! Thanks.